building-locally.md

```diff
@@ -0,0 +1,176 @@
+    [INFO] Installing /Users/user@ibm.com/BlueCompute/refarch-cloudnative-micro-orders/target/orders-1.0-SNAPSHOT.war to /Users/user@ibm.com/.m2/repository/projects/orders/1.0-SNAPSHOT/orders-1.0-SNAPSHOT.war
+    [INFO] Installing /Users/user@ibm.com/BlueCompute/refarch-cloudnative-micro-orders/pom.xml to /Users/user@ibm.com/.m2/repository/projects/orders/1.0-SNAPSHOT/orders-1.0-SNAPSHOT.pom
+    [INFO] ------------------------------------------------------------------------
+    [INFO] BUILD SUCCESS
+    [INFO] ------------------------------------------------------------------------
+    [INFO] Total time: 45.990 s
+    [INFO] Finished at: 2018-02-26T13:50:46-05:00
+    [INFO] Final Memory: 21M/250M
+    [INFO] ------------------------------------------------------------------------
+    ```
+
+**NOTE:** We will not setup RabbitMQ as that service is used to speak with other services, such as inventory.
+
+### Setting up MySQL
+
+To set up MySQL locally, we will need to run it in a docker container. 
+You need [Docker](https://www.docker.com/) as a prerequisite.
+
+1. To run MySQL on docker locally, run the below commands:
+
+    ```
+    $ cd mysql
+    ```
+
+2. Build the docker image.
+
+    ```
+    $ docker build -t mysql .
+    ```
+
+3. Run the container.
+
+    ```
+    $ docker run -p 9041:3306 -d --name mysql -e MYSQL_ROOT_PASSWORD=password mysql
+    ```
+
+
+4. It creates the `orders` table and loads the sample data.
+
+    ```
+    $ cd ..
+    ```
+
+### Setting up Zipkin 
+
+This is an optional step.
+
+In our sample application, we used Zipkin as our distributed tracing system.
+
+If you want to access the traces for orders service, run Zipkin as a docker container locally. 
+You can find the instructions and more details 
+[here](https://github.com/ibm-cloud-architecture/refarch-cloudnative-kubernetes/blob/microprofile/Zipkin/README.md)
+
+
+### Running the app and stopping it
+
+1. Setup environment variables.
+
+    Set the JDBC URL before you start your application. The host and port depends on the service you use. 
+    You can run the MySQL server locally on your system using the MySQL docker container or use the 
+    [MySQL Compose](https://www.ibm.com/cloud/compose/mysql) available in [IBM Cloud](https://www.ibm.com/cloud/).
+    
+    ```
+    $ export jdbcURL=jdbc:mysql://<Your host>:<Port>/ordersdb?useSSL=false
+    $ export dbuser=<DB_USER_NAME>
+    $ export dbpassword=<PASSWORD>
+    ```
+    
+    If you've been following along, your environment variables should be:
+        
+    ```
+    $ export jdbcURL=jdbc:mysql://localhost:9041/ordersdb?useSSL=false
+    $ export dbuser=root
+    $ export dbpassword=password
+    ```
+    
+    Also set the Zipkin host and port to defaults.
+    
+    ```
+    $ export zipkinHost=localhost
+    $ export zipkinPort=9411
+    ``` 
+
+1. Start your server.
+
+    ```
+    $ mvn liberty:start-server -DtestServerHttpPort=9081
```

Comment:
Can you please use a different port. I used the same in inventory. Can you use 9083?